spicy/toolchain/src/compiler/codegen/parser-builder.cc

```diff
@@ -1294,9 +1295,6 @@ hilti::type::Struct ParserBuilder::addParserMethods(hilti::type::Struct s, const
          builder::parameter("cur", type::Optional(type::stream::View()), builder::optional(type::stream::View())),
          builder::parameter("context", type::Optional(builder::typeByID("spicy_rt::UnitContext")))};
 
-    for ( auto p : t.parameters() )
-        params.emplace_back(p);
-
```

Comment:
Hmm, I don't think I see a replacement for this, don't we need this?

Reply:
This adds all unit parameters as parameters of the `parse*` functions. AFAICT this is not need as the parse methods either default-construct units (`parse1` or `parse3`), or work on already initialized units passed as parameters (`parse2`).

Since we previously disallowed units with parameters as entry points this code was up until now only called with an empty `parameters`.

Reply:
ok, see it now.